### DISCUSSION TITLE
Database opened in KeePassDX has missing entries, even if opened from synchronized Google Drive

### DISCUSSION THREAD

**Discussion by Krzeszny** [Kunzisoft/KeePassDX#1285] [Q&A] [answered]
This is very weird. I synchronize the database between my PC and smartphone with Google Drive but the database opened in DX has many less entries than it should have.

1. I triple-checked that the files on my PC and smartphone are synchronized (updated today). 
2. Both KeePassDX (Android) and XC (Windows) are updated.
3. I tried searching for the missing entries, as well as browsing database folders manually.
4. I opened the Google Drive file several times on DX.
5. I'm sure it's the same file.

The problem is that when I open my database from Google Drive in KeePassDX (downloaded from G-Droid), many entries are missing (whether I use the search function or browse the folders manually). The file is synchronized on Google Drive but KeePassDX is acting like it keeps opening a cached file despite me choosing the synchronized database from a file browser.

For instance: Root has 4 entries in DX while it should have 2. The first folder has 1 entry while it should have 2. Another folder has 2 entries too few, including one I added a week ago, etc.

I'd report it as an issue but I don't know why it's happening. I haven't tried clearing the cache yet. What's going on?

**Comment by J-Jamet** [comment, marked as answer]
Clear the cache of Google Drive.
The wiki explains it all very well and the procedure is well in there : https://github.com/Kunzisoft/KeePassDX/wiki/File-Manager-and-Sync
Same as #1215 #982
